frontend/test/metabase/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -52,39 +54,46 @@ describe("scenarios > embedding > smoke tests", () => {
       cy.location("pathname").should("eq", embeddingPage);
       cy.findByText("Enabled");
 
+      cy.findByText("Standalone embeds").click();
       if (isOSS) {
-        cy.findByText(/Customization/i);
-        cy.findByText(
-          "Looking to remove the “Powered by Metabase” logo, customize colors and make it your own?",
+        cy.contains(
+          "In order to remove the Metabase logo from embeds, you can always upgrade to one of our paid plans.",
         );
 
-        assertLinkMatchesUrl("Explore our paid plans.", upgradeUrl);
+        assertLinkMatchesUrl("paid plans.", upgradeUrl);
       }
 
       cy.findByText(/Embedding secret key/i);
       cy.findByText(
-        "Secret key used to sign JSON Web Tokens for requests to `/api/embed` endpoints.",
+        "Standalone Embed Secret Key used to sign JSON Web Tokens for requests to /api/embed endpoints. This lets you create a secure environment limited to specific users or organizations.",
       );
 
       getTokenValue().should("have.length", 64);
 
       cy.button("Regenerate key");
 
+      // List of all embedded dashboards and questions
+      cy.findByText(/Embedded dashboards/i);
+      cy.findByText("No dashboards have been embedded yet.");
+
+      cy.findByText(/Embedded questions/i);
+      cy.findByText("No questions have been embedded yet.");
+
       // Full app embedding section (available only for EE version and in PRO hosted plans)
       if (isEE) {
+        sidebar().within(() => {
+          cy.findByText("Embedding").click();
+        });
+        cy.findByText("Full-app embedding").click();
         cy.findByText(/Embedding the entire Metabase app/i);
         cy.contains(
-          "If you want to embed all of Metabase, enter the origins of the websites or web apps where you want to allow embedding in an iframe, separated by a space. Here are the exact specifications for what can be entered.",
+          "With this Pro/Enterprise feature you can embed the full Metabase app. Enable your users to drill-through to charts, browse collections, and use the graphical query builder. Learn more.",
+        );
+        cy.contains(
+          "Enter the origins for the websites or web apps where you want to allow embedding, separated by a space. Here are the exact specifications for what can be entered.",
         );
         cy.findByPlaceholderText("https://*.example.com").should("be.empty");
       }
-
-      // List of all embedded dashboards and questions
-      cy.findByText(/Embedded dashboards/i);
-      cy.findByText("No dashboards have been embedded yet.");
-
-      cy.findByText(/Embedded questions/i);
-      cy.findByText("No questions have been embedded yet.");
```

Comment:
I move this block up before the `if (isEE)` check because inside the `if (isEE)` it's testing a functionality on a different page (new UI)